src/app/shared/store/user.actions.ts

```diff
@@ -171,7 +171,7 @@ export class OnUpdateApplicationFail {
 }
 export class OnUpdateApplicationSuccess {
   static readonly type = '[user] update Application success';
-  constructor(public payload) { }
+  constructor(public payload: any) { }
```

Comment:
we don't use the typo "any", it's a bad practice. Add typo of AplicationUpdate, I believe this is the correct one.